BREAKING Overlay transaction support. (#3263)
paritytech/substrate: primitives/historical-data/README.md

## Historical data

Crate with functionality to manage data that stores its own history.

This covers:
- linear history driven data, eg. transactional layers for overlay.

> - long term storage with multiple branch, eg. offchain storage.

Reviewer: ```suggestion
- long term storage with multiple branches, eg. off-chain storage.
```

Reviewer: is that even the intended meaning?

Author: I am removing this line, it is something that is implemented in another PR, but since the code is currently not in the crate it does not make sense to keep it.

> General design is container where query and update requires global

Reviewer: ```suggestion
General design is to be a container where query and update both require a global
```

i'm changing this from non-english to english. i've no idea if what i'm writing is what you were meaning @cheme .

history context.

> History is serialize as a per item basis.

Reviewer: Not english.

> This crates is `no_std` compatible as long as the `std` feature is not enabled.

Reviewer: ```suggestion
This crate is `no_std` compatible as long as the `std` feature is not enabled.
```
